caesars-palace/icons/business-casual.svg

```diff
@@ -0,0 +1 @@
+<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 448 512" class="injected-svg" data-src="/content/dam/empire/icon-library/fontawesome-icons/user-tie.svg" xmlns:xlink="http://www.w3.org/1999/xlink"><path fill="#fff" d="M224 288c79.5 0 144-64.5 144-144S303.5 0 224 0 80 64.5 80 144s64.5 144 144 144zm0-240c52.9 0 96 43.1 96 96s-43.1 96-96 96-96-43.1-96-96 43.1-96 96-96zm91.9 256.2l-56.5 154.5L240 376l32-56h-96l32 56-19.5 82.7L132 304.2C58.9 305.5 0 365 0 438.4V464c0 26.5 21.5 48 48 48h352c26.5 0 48-21.5 48-48v-25.6c0-73.4-58.9-132.9-132.1-134.2zM96 464H48v-25.6c0-35.4 21.9-66.2 53-79.4l38.4 105H96zm304 0h-91.3L347 359c31 13.2 53 44 53 79.4V464z"></path></svg>
```

Comment:
We already have this one, it's https://github.com/hlxsites/caesars/blob/main/caesars-palace/icons/dress-code-smart-business-casual.svg. We should just rename it.

Reply:
Done!